check_ssl_cert

```diff
@@ -1786,6 +1859,19 @@ main() {
             if [ -n "${VERBOSE}" ] ; then
                 echo "  OCSP stapling enabled"
             fi
+            NEXT_UPDATE=$(grep -o 'Next Update: .*$' "${OCSP_RESPONSE_TMP}" | cut -b14-)
+
+	    days_until "${NEXT_UPDATE}"
+	    
+            OCSP_EXPIRES_IN_HOURS=$(( DAYS_UNTIL / 3600 ))
```

Comment:
The calculation of OCSP_EXPIRES_IN_HOURS is incorrect. From days you need to multiply by 24.
Another problem is that the days_until returns an int, so the hours goes in steps of 24. Preferably the method days_until is changed into hours_until (or even minutes_until), and the result is divided instead of multiplied.

Reply:
You are right, I'll take a look